platform/android/src/map_renderer.cpp

```diff
@@ -107,6 +106,13 @@ void MapRenderer::requestSnapshot(SnapshotCallback callback) {
     );
 }
 
+std::shared_ptr<Mailbox> MapRenderer::getMailbox() noexcept {
+    if (!mailbox) {
```

Comment:
The only way I see to protect `mailbox` to not be accessed directly, but only through `getMailbox()` would be to move it to a class and have`MailboxData::get()` and do a `mailboxData->get()` instead of `this->getMailbox()`. Perhaps overkill.

Reply:
Moved under MailboxData::getMailbox()